ur_ros2_control_demos/config/ur5_simulated_config.yaml

```diff
@@ -0,0 +1,68 @@
+###############################################
```

Comment:
I think this folder is an OK place to stick demo stuff for now. We'll clean it up later

Reply:
Please rename this file:  `ur5_servo_config.yaml`

Reply:
will do